test/io/testfilewriter.py

```diff
@@ -20,15 +20,30 @@
 
 class WriterTest(unittest.TestCase):
 
+    def setUp(self):
+        self.MissingAttributeError = cclib.io.filewriter.MissingAttributeError
```

Comment:
Removed this line. The exception is now being imported from the filewriter class https://github.com/cclib/cclib/pull/388/commits/b7a09f6219fb345918d942b6e2ca3baa58be1b3d#diff-b77b9f83bdfaba6902ee0d0d99d8ac7dR14